test/index.js

```diff
@@ -20,51 +19,153 @@ const orchestrator = new Orchestrator({
 
-  const result = await alice.call("myInstanceName", "holo_acorn", "fetch_goal", {"address": addr.Ok.goal.address})
-  console.log(addr)
+orchestrator.registerScenario("tow agent test create, update and archive goals ", async (s, t) => {
+  // the 'true' is for 'start', which means boot the Conductors
+  const {alice,bob,alex} = await s.players({alice: conductorConfig,bob: conductorConfig,alex:conductorConfig}, true)
+  // Make a call to a Zome function
+  // indicating the function, and passing it an input
+  const goal = await alice.call("acorn_hc", "holo_acorn", "create_goal", {"goal" : {"content":"sample content",
+ 'user_hash': alice._instances.acorn_hc.agentAddress,
+ "unix_timestamp":Date.now(),
+ "hierarchy": "Branch",
+ "status": "Uncertain"},"maybe_parent_address":null})
+ const goal2 = await bob.call("acorn_hc", "holo_acorn", "create_goal", {"goal" : {"content":"sample content",
+ 'user_hash': bob._instances.acorn_hc.agentAddress,
+ "unix_timestamp":Date.now(),
+ "hierarchy": "Branch",
+ "status": "Uncertain"},"maybe_parent_address":goal.Ok.goal.address})
+ 
+  // Wait for all network activity to
+  await s.consistency()
+  const time=Date.now()
+  const result_bob = await bob.call("acorn_hc", "holo_acorn", "update_goal", {"goal" : {"content":"sample content2",
+  'user_hash': alice._instances.acorn_hc.agentAddress,
+  "unix_timestamp":time,
+  "hierarchy": "Root",
+  "status": "Uncertain"},"address":goal.Ok.goal.address})
   // check for equality of the actual and expected results
-  t.deepEqual(result, { Ok: {"content":"sample content","user_hash": "HcSciJuZ4M4e4Iew7zsJCU8jdTxypahgfE5BJGzQANqhhmnjRa9vTu4snh44kuz","unix_timestamp":f,"hierarchy": "Branch","status": "Uncertain"}})
+  await s.consistency()
+  const result_alex=await alex.call("acorn_hc","holo_acorn","add_member_of_goal",{"goal_member":{"goal_address":goal.Ok.goal.address,
+    "agent_address":alice._instances.acorn_hc.agentAddress,"unix_timestamp":Date.now()}})
+    await s.consistency()
+  const result_alex2=await alex.call("acorn_hc","holo_acorn","fetch_goal_members",{})
+  const result_alice= await alice.call("acorn_hc", "holo_acorn", "fetch_goals",{})
+  t.isNotDeepEqual(goal2.Ok.maybe_edge,null)
+  t.deepEqual(result_alice.Ok.length,2)
+  t.deepEqual(result_bob.Ok.entry,{"content":"sample content2",
+  'user_hash': alice._instances.acorn_hc.agentAddress,
+  "unix_timestamp":time,
+  "hierarchy": "Root",
+  "status": "Uncertain"})
+  t.deepEqual(result_alex.Ok.entry,result_alex2.Ok[0].entry)
+  await s.consistency()
+  const result_alice2= await alice.call("acorn_hc", "holo_acorn", "archive_goal",{"address":goal.Ok.goal.address})
+  await s.consistency()
+  const result_alice3= await alice.call("acorn_hc", "holo_acorn", "fetch_edges",{})
+  const result_bob2= await bob.call("acorn_hc", "holo_acorn", "fetch_goals",{})
+  const result_alex3=await alex.call("acorn_hc","holo_acorn","fetch_goal_members",{})
+
+  t.deepEqual(result_alice2.Ok.address,goal.Ok.goal.address)
```

Comment:
If the things being compared are not JSON objects, you can just use `t.equal` instead of `t.deepEqual` ... see:
https://github.com/substack/tape#tequalactual-expected-msg
https://github.com/substack/tape#tdeepequalactual-expected-msg